test/SILOptimizer/ossa_rauw_tests.sil

```diff
@@ -302,52 +302,6 @@ bbExitBlock(%result : @owned $FakeOptional<Klass>):
   return %result : $FakeOptional<Klass>
 }
 
-// CHECK-LABEL: sil [ossa] @unowned_to_guaranteed_rauw_2 : $@convention(thin) (@guaranteed Klass) -> (Klass, Klass) {
```

Comment:
I think these test cases are also OK to go, specifically testing the cases that we never generate. And we have versions with @owned.